third_party/terraform/utils/provider.go.erb

```diff
@@ -228,10 +228,11 @@ func Provider() *schema.Provider {
 			"google_project":                                   dataSourceGoogleProject(),
 			"google_projects":                                  dataSourceGoogleProjects(),
 			"google_project_organization_policy":               dataSourceGoogleProjectOrganizationPolicy(),
-			"google_secret_manager_secret_version":              dataSourceSecretManagerSecretVersion(),
+			"google_pubsub_topic":                              dataSourceGooglePubsubTopic(),
```

Comment:
Can you also push this change (putting it in the right place in the list) in the original PR so that we don't potentially have merge conflicts?

Reply:
(I have no idea whether this is an actual problem that we could get. I usually just try to be cautious about things that I wouldn't discover until after-the-fact. Nathan would know for sure)

Reply:
done